astropy/cosmology/tests/test_flrw.py

```diff
@@ -29,6 +29,8 @@
 from .test_core import CosmologySubclassTest as CosmologyTest
 from .test_core import FlatCosmologyMixinTest, ParameterTestMixin, invalid_zs, valid_zs
 
+__doctest_skip__ = ['ellipkinc', 'hyp2f1']
```

Comment:
Test file too? I don't know how I feel about this. I feel like we are abusing `__doctest_skip__`.

Reply:
Well, there is nothing about a ufunc that you can find out by introspection to determine the module it was declared in.

Reply:
I get that a ufunc can't be introspected, but how does pytest-doctestplus pick up the test file in the first place? Shouldn't they skipped because they aren't doctested?

Reply:
Those two ufuncs do have doctests in their docstrings.

Reply:
Sorry for the confusion, I meant I thought `pytest-doctestplus` filtered out test files an initio, but I guess that's not the case?

Reply:
Come again? I'm sorry, I didn't understand your question.

Reply:
I presumed when `pytest-doctestplus` was assembling the list of docstrings it needed to check, it would skip over test files. E.g.

```
module/
    file.py
    tests/  # skipped
        test_file.py  # skipped, so it doesn't matter if there's a ufunc inside
```

Reply:
Oh, that's probably a good idea. But I doubt that even pytest does that by default, does it?